nix/modules/kld/default.nix

```diff
@@ -178,7 +185,9 @@ in
       ];
     };
 
-    networking.firewall.allowedTCPPorts = lib.optionals cfg.openFirewall [ 9234 ];
+    networking.firewall.allowedTCPPorts = [ ]
+      ++ lib.optionals cfg.openFirewall [ cfg.peerPort ];
+    networking.firewall.extraCommands = lib.concatMapStrings (ip: "iptables -A nixos-fw -p tcp --source " + ip + " --dport ${toString cfg.restApiPort} -j nixos-fw-accept || true") cfg.apiIpAccessList;
```

Comment:
This will break if the list contains an ipv6 address since iptables only works for ipv4 addresses.

Reply:
```suggestion
    networking.firewall.extraCommands = lib.concatMapStrings (ip:
      if lib.hasInfix ":" ip then ''
        ip6tables -A nixos-fw -p tcp --source " + ip + " --dport ${toString cfg.restApiPort} -j nixos-fw-accept
    '' else ''
        iptables -A nixos-fw -p tcp --source " + ip + " --dport ${toString cfg.restApiPort} -j nixos-fw-accept
    '') cfg.apiIpAccessList;
```